src/test/scala/nl/knaw/dans/easy/solr/ConfSpec.scala

```diff
@@ -0,0 +1,107 @@
+package nl.knaw.dans.easy.solr
```

Comment:
Please, try not to go overboard with unit testing things. Focus on unit testing the core functionality. We don't want to just create more code to maintain.